src/Test/Utilities/Portable/Compilation/ControlFlowGraphVerifier.cs

```diff
@@ -783,10 +783,14 @@ PooledHashSet<CaptureId> adjustAndGetEntryState(ArrayBuilder<PooledHashSet<Captu
                     Assert.True(state.Contains(id) || isCaptureFromEnclosingGraph(id),
                         $"Operation [{operationIndex}] in [{getBlockId(block)}] uses not initialized capture [{id.Value}].");
 
+                    // Except for a few specific scenarios, any references to captures should either be long-lived capture references,
+                    // or they should come from the enclosing region. Otherwise, it signifies that the region packing algorithm failed
```

Comment:
>it signifies that the region packing algorithm failed [](start = 81, length = 53)

If that refers to the pack pass at the end of the graph building process, I think this is not accurate. If I remember correctly, it is not responsible for enforcing "proper" distribution of captures among regions.